.github/workflows/ci.yml

```
@@ -24,30 +24,41 @@ jobs:
           distribution: 'temurin'
           java-version: ${{ matrix.java-version }}
 
-      # Purge cached dependencies to avoid stale SNAPSHOTs
-      - name: Purge local repository
-        run: mvn dependency:purge-local-repository -DreResolve=true -B
-
-      - name: Cache local Maven repository
+      - name: Cache Gradle dependencies
         uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)